library/core/class.dispatcher.php

```diff
@@ -720,7 +720,11 @@ private function rewriteRequest($request) {
         } elseif (in_array($request->path(), ['', '/'])) {
             $this->isHomepage = true;
             $defaultController = Gdn::router()->getRoute('DefaultController');
+            $originalGet = $request->get();
             $request->pathAndQuery($defaultController['Destination']);
+            if (is_array($originalGet) && !empty($originalGet)) {
```

Comment:
Nitpicking: Since we know that the variable is an array we should probably use `count()`